lib/ioc-rc

```diff
@@ -543,10 +543,12 @@ __rc_jails () {
         local name="$(zfs get -H -o value org.freebsd.iocage:host_hostuuid \
                     $jail)"
         local boot="$(zfs get -H -o value org.freebsd.iocage:boot $jail)"
+        local _jail_hostid="$(zfs get -H -o value org.freebsd.iocage:hostid \
+                    $jail)"
         local priority="$(zfs get -H -o value org.freebsd.iocage:priority \
-                        $jail)"
```

Comment:
Yep you are correct :) GitHub's diff just looked off until I viewed the whole file.